src/segments/helm.go

```diff
@@ -0,0 +1,57 @@
+package segments
+
+import (
+	"github.com/jandedobbeleer/oh-my-posh/src/platform"
+	"github.com/jandedobbeleer/oh-my-posh/src/properties"
+)
+
+type Helm struct {
+	props properties.Properties
+	env   platform.Environment
+
+	Version string
+}
+
+const (
+	// NewProp enables something
+	NewProp properties.Property = "newprop"
+)
+
+func (h *Helm) Enabled() bool {
+	displayMode := h.props.GetString(DisplayMode, DisplayModeAlways)
+	if displayMode != DisplayModeFiles {
+		return h.getVersion()
+	}
+
+	_, erryml := h.env.HasParentFilePath("Chart.yml")
```

Comment:
I would rewrite this with a for loop so it's extensible easily.

```golang
files := []string{"Chart.yml", "Chart.yaml"}
var found bool
for _, file := range files {
    if _, err  := h.env.HasParentFilePath("Chart.yaml"); err != nil {
        found = true
        break
    }
}

if !found {
    return false
}
```

Reply:
Thank you. Fixed!